telemetry.go

```diff
@@ -25,6 +25,12 @@ var (
 		},
 		[]string{"type"},
 	)
+	eventsFlushed = prometheus.NewCounter(
+		prometheus.CounterOpts{
+			Name: "statsd_exporter_events_flushed_total",
```

Comment:
I would like to make the metric name less ambiguous:

```suggestion
			Name: "statsd_exporter_event_queue_flushed_total",
```

the help makes it clear but is not really visible in Prometheus, so I think it's important to be clear that this is not a count of events but a count of flushes.